src/js/components/DocumentButton.jsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+function handleClick(onClick, event, link) {
+  event.preventDefault();
+  const newWindow = window.open('', '_blank');
+  onClick()
+    .then(() => { newWindow.location.href = link; });
```

Comment:
You dont need to add a const for new window (from docs:
window.open(url, windowName, [windowFeatures]);
)